.gitguardian.example.yml

```diff
@@ -36,3 +36,15 @@ secret:
   # Detectors to ignore.
   ignored-detectors: # default: []
     - Generic Password
+
+iac:
+  # Exclude files and paths by globbing
+  ignored-paths:
+    - 'tests/'
+
+  # Ignore IaC vulnerability types by their id
+  ignored-policies:
+    - GG_IAC_0000
```

Comment:
Better to have an example with at least two ignored policies to show it is possible